### PR TITLE
Initial documentation for the Rotel Media Player platform

### PR DESCRIPTION
**Pull request by pp81381**
**Description:**

Initial documentation for the Rotel Media Player platform

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23692

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/rotel.markdown [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+layout: page
+title: "Rotel Media Player"
+description: "Rotel Media Player platform implementation.   Currently only supports the Rotel RSP-1570 Processor."
+date: 2019-04-27 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rotel.svg
+ha_release: 0.93.0
```
Comment (klaasnicolaas) [inline source/_components/rotel.markdown]:
```suggestion
ha_release: 0.93
```

File: source/_components/rotel.markdown [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: "Rotel Media Player"
+description: "Rotel Media Player platform implementation.   Currently only supports the Rotel RSP-1570 Processor."
```
Comment (klaasnicolaas) [inline source/_components/rotel.markdown]:
```suggestion
description: "Rotel Media Player platform implementation."
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/rotel.markdown [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+footer: true
+logo: rotel.svg
+ha_release: 0.93
+ha_category: Media Player
+ha_iot_class: Local Push
+---
+
+A Media Player platform that can control a Rotel Amplifier or Processor.
+
+At the moment the platform only supports the Rotel RSP-1570 processor but the underlying third party library could be adapted to support other devices of the same vintage.
```
Comment (frenck) [inline source/_components/rotel.markdown:18]:
```suggestion
At the moment the platform only supports the Rotel RSP-1570 processor but the underlying third-party library could be adapted to support other devices of the same vintage.
```

File: source/_components/rotel.markdown [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+name:
+  description: Name of the device.
+  required: false
+  default: Rotel RSP-1570
+  type: string
+source_aliases:
+  description: Mapping of default sources to aliases. The default sources are `' CD'`, `TAPE`, `TUNER`, `MULTI`, `VIDEO 1`, `VIDEO 2`, `VIDEO 3`, `VIDEO 4`, `VIDEO 5`. Each key must exactly match a default source name. The values must exactly match the labels configured in the device.  An empty alias indicates that the source is not used. A missing alias means that the default source should appear as-is in the source list.
+  required: false
+  default: {}
+  type: dict
```
Comment (frenck) [inline source/_components/rotel.markdown:57]: `dict` is not a valid type for our documentation. Please see:

<https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration>

File: source/_components/rotel.markdown [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+Some examples might be:
+
+* `/dev/ttyUSB0` (Linux)
+* `COM3` (Windows)
+* `socket://192.168.0.100:50000` (if you are using a TCP/IP to serial  converter)
+
+## {% linkable_title Services %}
+
+The media player component offers the followiing services:
```
Comment (frenck) [inline source/_components/rotel.markdown:72]:
```suggestion
The media player component offers the following services:
```

File: source/_components/rotel.markdown [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+Service Name | Parameters | Description
+-------------|------------|------------
+`rotel_send_command`|`entity_id`, `command_name`|Send a command to the media player.   See `commands.py` ([here](https://github.com/pp81381/rsp1570serial/blob/master/rsp1570serial/commands.py)) in the [rsp1570serial](https://github.com/pp81381/rsp1570serial) GitHub project for a full list of available commands.
+`rotel_reconnect`|`entity_id`|Reconnect to the media player
+
+The `entity_id` parameter can be a single entity id, a comma separated list or the word `all`.
```
Comment (frenck) [inline source/_components/rotel.markdown:79]:
```suggestion
The `entity_id` parameter can be a single entity id, a comma-separated list or the word `all`.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @pp81381, I've reviewed your PR and left some comments. Nothing big, don't worry!
Could you please take a look? Thanks already! 👍

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**